tools/font-edit/twin-fedit.h

```diff
@@ -23,11 +23,8 @@
 #ifndef _TWIN_FEDIT_H_
 #define _TWIN_FEDIT_H_
 
-#include <X11/Xatom.h>
-#include <X11/Xlib.h>
-#include <X11/Xutil.h>
-#include <X11/keysym.h>
-#include <cairo-xlib.h>
+#include <SDL2/SDL.h>
```

Comment:
Don't include SDL2 header here. Always include what you exactly need.

Reply:
Resolve in 7231d62d0106fb56594ddd7785efcc9ab9779b04

SDL2 header was moved to twin-fedit.c.